tests/DCPS/CompatibilityTest/run_test.pl

```diff
@@ -20,6 +20,7 @@
 my $is_rtps_disc = 0;
 my $DCPScfg = "";
 my $level = 4;
+my $ll = "info";
```

Comment:
When `DCPSDebugLevel` is set above 1, the log level is set to debug, which includes info, so I don't think adding this is necessary.